### ISSUE TITLE
Allowing to send test emails in ps_reminder

### ISSUE THREAD

**Issue by PrestaShark** [PrestaShop/PrestaShop#18569]
Hi,
ps_reminder module need to get new feature to send test reminders to store admin or selected email.
For now its hard to fully test functionality on production store.

Thanks!

**Comment by florine2623** [comment]
Hi @PrestaShark ,
Thanks for the recommandation, I will let our Product Team know your request.
Thanks !

**Comment by florine2623** [comment]
Hi @PrestaShop/prestashop-product-team !

What do you think about this feature regarding free _Customer follow-up_ module ?
latest version 2.0.0

Thanks !

**Comment by colinegin** [comment]
Yes it is a good idea ! Have you already created a PR your side @PrestaShark ?
Thanks !
